app/services/symphony_client.rb

```diff
@@ -34,10 +34,14 @@ def login_by_sunetid(sunetid)
 
   # get a session token by authenticating to symws
   def session_token
-    @session_token ||= begin
+    @session_token ||=
+      begin
       response = request('/user/staff/login', json: Settings.symws.login_params, method: :post)
 
       JSON.parse(response.body)['sessionToken']
+      rescue JSON::ParserError
+        Honeybadger.notify('Unable to connect to Symphony Web Services.')
```

Comment:
@shelleydoljack @jkeck Do we want to add a Honeybadger config to ignore these notifications? The purpose would  only be to have it logged for reference if we need it for troubleshooting or follow-up.

Reply:
I would leave it up to you all as to if that is something you care about being notified about.  It can always be removed or added later, just wanted to raise the question (as this will no happen completely w/o our knowledge once we are rescuing).

Reply:
I guess I am confused about the difference between ignoring a Honeybadger alert by using the configuration, and ignoring an alert (i.e. `is:ignored`) in the Honeybadger interface. I assumed these were the same thing, but they are not(?)

I think we should just go ahead and get the notifications for the time being.